Dockerfile.bolt-server

```diff
@@ -10,7 +10,7 @@ RUN mkdir /bolt-server
 # Gemfile requires gemspec which requires bolt/version which requires bolt
 ADD . /bolt-server
 WORKDIR /bolt-server
-RUN bundle install --no-cache --path vendor/bundle
+RUN bundle install --no-cache --path vendor/bundle --jobs=$(nproc)
```

Comment:
This will conflict with #3083 but :+1: for the change.

Reply:
yes, I will rebase one after the merge. I thought splitting it up into multiple PRs gives a better history.